docs/development/extensions-core/druid-basic-security.md

```diff
@@ -62,16 +69,16 @@ druid.auth.authenticator.MyBasicMetadataAuthenticator.authorizerName=MyBasicMeta
 ```
 
 To use the Basic authenticator, add an authenticator with type `basic` to the authenticatorChain.
-The authenticator needs to also define a credentialsValidator with type 'metadata' or 'ldap'.
-If credentialsValidator is not specified, type 'metadata' will be used as default.
+The authenticator must define a credentials validator `credentialsValidator` with type 'metadata' or 'ldap'.
+Otherwise the credentials validator defaults to `metadata` if you do not specify another type.
```

Comment:
above 2 lines seems to be contradictory, first one says you must define it and 2nd mentions a default value. 

probably, reword as - "The authenticator uses metadata credentials validator by default which can be configured to `metadata`." Or similar wordings.

Reply:
@nishantmonu51 , @suneet-s updated this to reflect that `metadata` is the default. And you only need to define one for `LDAP`. PTAL. Not sure if I've 100% got the nuance here.